src/main/java/org/opensearch/neuralsearch/processor/NormalizationProcessorWorkflow.java

```diff
@@ -103,6 +104,11 @@ public void execute(final NormalizationProcessorWorkflowExecuteRequest request)
         );
     }
 
+    private boolean getIsSingleShard(final NormalizationProcessorWorkflowExecuteRequest request) {
```

Comment:
There is a method in this class getFromValueIfSingleShard in that we check for whether it is a single shard. Can we replicate the same here and also call isSingleShard method there as well?